components/ai_chat/core/browser/conversation_driver.cc

```diff
@@ -1013,6 +1020,15 @@ bool ConversationDriver::HasPendingConversationEntry() {
   return !pending_conversation_entry_.is_null();
 }
 
+void ConversationDriver::StopGeneration() {
+  ClearAllQueries();
```

Comment:
If StopGeneration is called after a user entry is added to history but before any assistant entry is created, do subsequent messages still work (on both conversation API and existing API)?

Reply:
<img width="319" alt="image" src="https://github.com/brave/brave-core/assets/8665427/869a07c7-c00f-4a1d-bc0c-b320b9bc501a">

works ok